chef/cookbooks/ceph/recipes/osd.rb

```diff
@@ -71,7 +71,7 @@
   if is_crowbar?
     dirty = false
 
-    node.set["ceph"]["osd_devices"] ||= []
+    node.set["ceph"]["osd_devices"] = [] if node["ceph"]["osd_devices"].nil?
```

Comment:
wow, great finding @scottwulf !

Im ok with this change, seems better than the push.